src/components/portal/pages/reviews/Home.tsx

```diff
@@ -68,22 +108,37 @@ export default class ReviewsHome extends PortalPageBase {
       // TODO: Add two-three proposals to review for two different users.
       // TODO: At least one of these proposals should have both users as reviewer
       // TODO: Should be able to show "completed reviews" for this shared proposal
-    const proposals = this.getProposalList();
+    const reviews = this.getReviewList();
     const content:JSX.Element[] = [];
-    proposals.forEach((proposal) => {
+    reviews.forEach((review) => {
+      console.log('107, reviews/home', review);
       content.push(
         <tr>
-          <td>{proposal.id}</td>
-          <td>{proposal.title}</td>
+          <td>{review.proposalTitle}</td>
+          <td>{moment().add(((Math.floor(Math.random() * 7) + 3)), 'days').format('MM-DD-YYYY')}</td>
```

Comment:
I'm not sure I understand what this is... Is it a placeholder?

Reply:
Yes, the creation of Reviews is a placeholder at the moment. The creation of them is handled on the EUSAdmin side of the application. Reviews will be stored in the DB like Proposals and once we have a server to query against, the reviews will just query for all the reviews that the current user has open.